src/nvim/ex_docmd.c

```diff
@@ -9756,6 +9805,9 @@ Dictionary commands_array(buf_T *buf)
     PUT(d, "bar", BOOLEAN_OBJ(!!(cmd->uc_argt & EX_TRLBAR)));
     PUT(d, "register", BOOLEAN_OBJ(!!(cmd->uc_argt & EX_REGSTR)));
     PUT(d, "keepscript", BOOLEAN_OBJ(!!(cmd->uc_argt & EX_KEEPSCRIPT)));
+    PUT(d, "preview", BOOLEAN_OBJ(!!(cmd->uc_argt & EX_PREVIEW)));
+    PUT(d, "preview_arg", cmd->uc_preview_arg == NULL
```

Comment:
why are both params needed? when would one be set but not the other?

Reply:
> why are both params needed? when would one be set but not the other?

When "preview" is a Luaref